ciao-image/main.go

```diff
@@ -31,6 +32,8 @@ var identity = "https://localhost:35357/"
 var userName = "csr"
 var password = "hello"
 var mountPoint = "/var/lib/ciao/images"
+var dbDir = "/var/lib/ciao/images"
```

Comment:
We also need to modify singlevm to clean up this.  This should be done in setup.sh around line 119 after we delete the controller.db file.